app/mobile/src/components/LandingScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { StyleSheet, Text, View, Image } from "react-native";
+import LandingImage from "../assets/landing-bg.png";
+import { Dimensions } from "react-native";
+import LargeButton from "./atoms/LargeButton";
+
+const windowWidth = Dimensions.get("window").width;
+const windowHeight = Dimensions.get("window").height;
```

Comment:
Might want to move this inside the screen component (function) before the JSX view stuff, so it gets re-computed on screen orientation.

Reply:
to be resolved today by tomas by tyranically locking us to portrait mode.

Reply:
on review, we are already locked to portrait in app.json. resolving.